src/components/Search.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+function Search (props) {
```

Comment:
could be rewritten with arrow function

`const Search = () => `

Reply:
Arrow Functionality implemented